src/lib/storage/fake_device_factory.rb

```diff
@@ -242,7 +242,7 @@ def create_partition_table(parent, args)
       # @return [Fixnum]
       def str_to_ptable_type(string)
         # Allow different spelling
-        string = "msdos" if string.downcase == "ms-dos"
+        string = "msdos" if string.casecmp("ms-dos").zero?
```

Comment:
On 19.07.2016 14:30, Ladislav Slezák wrote:
>> -        string = "msdos" if string.downcase == "ms-dos"
>> > +        string = "msdos" if string.casecmp("ms-dos").zero?
>
> But there is one important thing: if you cannot remember the exact
 > syntax for the new style comparison you can still use the old way
 > (`foo.downcase == "bar"`) and let `rubocop -a` fix it for you.

I would not consider that a "fix": It auto-replaces something readable with 
something that is hard to understand.


Kind regards
-- 
Stefan Hundhammer  <shundhammer@suse.de>
YaST Developer

SUSE Linux GmbH
GF: Felix Imendörffer, Jane Smithard, Graham Norton; HRB 21284 (AG Nürnberg)
Maxfeldstr. 5, 90409 Nürnberg, Germany